src/platforms/python/common/troubleshooting.mdx

```diff
@@ -114,3 +115,58 @@ be fixed from within the SDK.
 This [issue has been fixed with gevent
 20.5](https://github.com/gevent/gevent/issues/1407) but continues to be one for
 eventlet.
+
+## Django Channels
```

Comment:
This should not be in common Python. Perhaps we can abuse PlatformContent and create a new type of platform content called python-troubleshooting-extra?

@dcramer wdyt

Reply:
I will try a PlatformSection here.